### ISSUE TITLE
Settings don't seem to be saved?

### ISSUE THREAD

**Issue by ahayzen** [flathub/org.armagetronad.ArmagetronAdvanced#3]
Every time I launch the game it asks for the first and second language, it seemingly it's successfully storing this setting (same happens for other setttings like controls, fullscreen mode).

I see in the CLI there a load of different directories that can be given as arguments, wonder one of these needs to be set so it stores the config in `~/.var/app/org.armagetronad.ArmagetronAdvanced` somewhere or something.
```
--datadir <Directory>        : read game data (textures, sounds and texts)
                               from this directory
--userdatadir <Directory>    : read customized game data from this directory
--configdir <Directory>      : read game configuration (.cfg-files)
                               from this directory
--userconfigdir <Directory>  : read user game configuration from this directory
--vardir <Directory>         : save game logs and highscores in this directory
```
Note that I don't spot any errors in the console of the app.

**Comment by TingPing** [comment]
Patches welcomed I suppose.

Alternatively you can run with `--filesystem=home`, see where it saved, and add `--persist=.armegetron` or whatever.

**Comment by zmanuel** [comment]
Fixed for 0.2.9.1.0 by forcing the game to use the XDG standard directories for its persistent data.
